pkg/descheduler/strategies/duplicates.go

```diff
@@ -79,8 +84,13 @@ func RemoveDuplicatePods(
 
 	evictable := podEvictor.Evictable(evictions.WithPriorityThreshold(thresholdPriority))
 
+	duplicatePods := make(map[podOwner]map[string][]*v1.Pod)
+	ownerKeyOccurence := make(map[podOwner]int32)
+	nodeCount := 0
+	nodeMap := make(map[string]*v1.Node)
+
 	for _, node := range nodes {
-		klog.V(1).InfoS("Processing node", "node", klog.KObj(node))
+		klog.V(0).InfoS("Processing node", "node", klog.KObj(node))
```

Comment:
why change this?

Reply:
leftover from running unit tests, sorry, undone